test/util.js

```diff
@@ -2,7 +2,7 @@
 
 jest.mock('cross-fetch');
 import Alpha from '../';
-const alpha = Alpha();
+const alpha = Alpha({ key: process.env.API_KEY });
```

Comment:
Lets keep the old env name

```suggestion
const alpha = Alpha({ key: process.env.AV_KEY });
```